ios/FluentUI/Tab Bar/TabBarItem.swift

```diff
@@ -90,6 +114,19 @@ open class TabBarItem: NSObject {
     /// Notification sent when the tab bar item's badge value changes.
     static let badgeValueDidChangeNotification = NSNotification.Name(rawValue: "TabBarItemBadgeValueDidChangeNotification")
 
+    /// Notification sent when item's `isUnread` value changes.
+    static let isUnreadValueDidChangeNotification = NSNotification.Name(rawValue: "TabBarItemisUnreadValueDidChangeNotification")
+
+    /// This value will determine whether or not to show the mark that represents the "unread" state.
+    /// The default value of this property is false.
+    @objc public var isUnread: Bool = false {
```

Comment:
would the better naming be isUnreadDotVisible ? because you can set badge value which also means something is unread yet.